project/SparkBuild.scala

```diff
@@ -968,6 +973,13 @@ object Volcano {
   )
 }
 
+object YuniKorn {
+  // Exclude all yunikorn file for Compile and Test
+  lazy val settings = Seq(
+    unmanagedSources / excludeFilter ~= { _ || "*YuniKorn*.scala" }
```

Comment:
```
unmanagedSources / excludeFilter ~= { _ || "*YuniKorn*.scala" }
```
which is equivalent to:
```
unmanagedSources / excludeFilter <<=
(unmanagedSources / excludeFilter) { (currentFilter: FileFilter) =>
currentFilter || "*YuniKorn*.scala"
}
```

[1] https://groups.google.com/g/simple-build-tool/c/zVMyoWRAVWg?hl=en

Note for myself and othre reviewers. : )